lib/helpers.bash

```diff
@@ -115,6 +118,27 @@ _bash-it-plugins ()
     _bash-it-describe "plugins" "a" "plugin" "Plugin"
 }
 
+_bash-it_update() {
+  _about 'updates Bash it'
+  _group 'lib'
+
+  cd "${$BASH_IT}"
+  git fetch &> /dev/null
+  local status="$(git rev-list master..origin/master 2> /dev/null)"
+  if [[ -n "${status}" ]]; then
+    git pull --rebase &> /dev/null
+    if [[ $? -eq 0 ]]; then
+      echo "Bash it successfully updated, enjoy!"
```

Comment:
Just a nitpick, but it looks like we have a range of spellings for the name of this project:

* Bash it
* Bash-it
* bash-it
* bash_it

This might be the time when we want to decide on one spelling. I thought that we had pretty much decided on "Bash-it" in #650 - didn't we?